### PR TITLE
Add PyPi description

### PR DESCRIPTION
**Pull request by gadhagod**
It now has a description on pypi. Before, this is what you saw on PyPi's description:

> The author of this package has not provided a project description

Now it should have the README.rst contents.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
 from setuptools import setup
 
+with open('README.rst', 'r') as fh:
+      long_description = fh.read()
+
 setup(name='schoolopy',
-      version='0.2.0',
+      version='0.2.2',
```
Comment (ErikBoesen) [inline setup.py:9]: Why 0.2.2 and not 0.2.1?

Comment (gadhagod) [reply setup.py:9]: Because the project description will update only if you create a new release.

**Comment by gadhagod** [comment]
The current release is 0.2.1

**Comment by ErikBoesen** [comment]
I wonder how I managed to push a 0.2.1 release to PyPI and forgot to commit it. Well, no matter anyway.

**Pull request merged**